### PR TITLE
feat: Added Code Snippets

### PR DESCRIPTION
**Pull request by LeAlencar**
Js Examples to consume every post request that is available today.

### CODE REVIEW

**Review by daniloab — COMMENTED** [review]
can you place a print?

File: src/swaggers/openpix.yml [hunk 1]
```diff
@@ -243,41 +248,56 @@ paths:
             return_existing:
               value: true
       description: Endpoint to create a new Charge for a customer
+      x-codeSamples:
```
Comment (sibelius) [inline src/swaggers/openpix.yml:251]: image

can we autogenerate this?

**Comment by LeAlencar** [comment]
Forgot about the print, lol. 
Every Request samples has a Javascript toggle now.
Here's how it looks:
image

**Comment by daniloab** [comment]
> Forgot about the print, lol. Every Request samples has a Javascript toggle now. Here's how it looks: image

Awesome, thanks!

How can we put the header Authorization, required for all requests, in the example?

**Comment by LeAlencar** [comment]
> > Forgot about the print, lol. Every Request samples has a Javascript toggle now. Here's how it looks: image
> 
> Awesome, thanks!
> 
> How can we put the header Authorization, required for all requests, in the example?

Putting it above the Accept param is enough?

**Comment by daniloab** [comment]
> > > Forgot about the print, lol. Every Request samples has a Javascript toggle now. Here's how it looks: image
> > 
> > 
> > Awesome, thanks!
> > How can we put the header Authorization, required for all requests, in the example?
> 
> Putting it above the Accept param is enough?

Yes

**Comment by LeAlencar** [comment]
@sibelius @daniloab  added auto-generated code samples and created a command to generate new ones if needed. 
https://github.com/cdwv/oas3-api-snippet-enricher and https://github.com/ErikWittern/openapi-snippet where used to do it.

Here's how it looks now.

image

**Comment by daniloab** [comment]
> @sibelius @daniloab added auto-generated code samples and created a command to generate new ones if needed. https://github.com/cdwv/oas3-api-snippet-enricher and https://github.com/ErikWittern/openapi-snippet where used to do it.
> 
> Here's how it looks now.
> 
> image

Thanks again, @LeAlencar 

can you put these instructions into the readme for the next contributors?

**Review by sibelius — APPROVED** [review]
awesome contribution

**Pull request merged**